script/mafia

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh -eu
 
+MAFIA_HOME=${MAFIA_HOME:-$HOME/.mafia}
+
```

Comment:
"better" is relevant, as @erikd-ambiata said, this will work. The only other one that you will see lots in scripts is:

```
: ${MAFIA_HOME:=$HOME/.mafia}
```

It isn't better/worse, just pointing it out because it occurs a fair bit.